.npmignore

```diff
@@ -1 +1,17 @@
-node_modules
\ No newline at end of file
+# Source code
+api/
+beautify/
+minify/
+tests/
+serviceWorker.txt
+*.ts
+tsconfig.json
+
+# Node modules
+node_modules/
+
+# Project configuration
+.appveyor.yml
+.eslintrc.json
```

Comment:
Is `.eslintrc.json` a project configuration or is it part of the package? If part of the package, my mistake and I'll remove it.

Reply:
I do not understand the distinction.  They are both configuration files.  The appveyor file defines the build steps for the appveyor Windows continuous integration service.

The eslintrc file defines the configuration for executing ESLint on the project.  This where I define my opinions and subjective thoughts on code style and I refuse to set or define any other subjectivity.  I am not a believer in code style or enforcing subjective opinions on people, but consistency is important.

Reply:
Oops, I should clarify: I'm asking whether or not the `.eslintrc.json` file should be included in the published module. My guess was project configuration for those working on the source code, but I wanted to be sure given the global dependency on eslint. Totally support helping contributors keep a consistent style in your codebase 💯

Reply:
The .eslintrc.json file is still helpful in the NPM package, because the *test* and *lint* commands will continue to execute if ESLint is installed on the user's system.  The other configuration files don't matter to the NPM package though.